maps/RandomZLevels/wildwest_props.dm

```diff
@@ -0,0 +1,324 @@
+	processing_objects -= src
+	..()
+
+/obj/structure/flora/desert/tumbleweed/process()
+	if(prob(98))
+		return
+	throw_at(get_turf(pick(orange(7,src))), 10,2)
+
+/obj/structure/sarcophagous
+	name = "sarcophagous"
+	desc = "Although often associated with Egyptians, sarchopagous is a Greek word meaning 'eater of flesh'. It refers to any stone burial recepticle."
+	density = TRUE
+	anchored = TRUE
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "morguestone"
+
+///////////////Effects//////////////
+
+/obj/effect/floating_candle
+	name = "floating candle"
+	desc = "The ghost of a candle? This is extremely cursed."
+	icon = 'icons/obj/candle.dmi'
+	icon_state = "floatcandle"
+	anchored = TRUE
+
+/obj/effect/floating_candle/New()
+	..()
+	set_light(8, 4, LIGHT_COLOR_CYAN)
+
+/obj/effect/tractorbeam
+	name = "tractor beam"
+	desc = "???"
+	icon = null
+	icon_state = null
+	anchored = TRUE
+	density = TRUE
+	var/turf/endpoint
+
+/obj/effect/tractorbeam/New()
+	..()
+	set_light(4, 8, LIGHT_COLOR_HALOGEN)
+	for(var/obj/effect/landmark/L in landmarks_list)
+		if(L.name == "tractor beam")
+			endpoint = get_turf(L)
+			break
+
+/obj/effect/tractorbeam/Bumped(atom/movable/AM)
+	AM.forceMove(endpoint)
+	if(ismob(AM))
+		AM << 'sound/music/xfiles.ogg'
+		to_chat(AM, "<span class='warning'>Gravity seems to lapse as you float into the sky!</span>")
+
+//Landmarks
+/obj/effect/landmark/respawner/desert
+	name = "Wild West respawner"
+
+///Spells
+/mob/living/proc/mountup()
+	for(var/spell/mountup/M in spell_list)
+		M.cast(src,src)
+		return
+
+/spell/mountup
+	name = "Mount Up"
+	desc = "Mount a steed."
+	charge_max = 0
+	spell_flags = 0
+	cast_delay = 2 SECONDS
+	var/obj/effect/overlay/my_overlay
+	var/active = FALSE
+	var/remembered_speed
+
+/spell/mountup/New()
+	..()
+	my_overlay = new /obj/effect/overlay/horsebroom_mount
+
+/spell/mountup/choose_targets(var/mob/user = usr)
+	return list(user)
+
+/spell/mountup/cast(var/list/targets, var/mob/user)
+	if(!active)
+		var/choosefile = pick('sound/items/jinglebell1.ogg','sound/items/jinglebell2.ogg','sound/items/jinglebell3.ogg')
+		playsound(user, choosefile, 100, 1)
+		user.register_event(/event/damaged, src, .proc/dismount)
+		user.overlays.Add(my_overlay)
+		active = TRUE
+		if(istype(user,/mob/living/simple_animal))
+			var/mob/living/simple_animal/SA = user
+			remembered_speed = SA.speed
+			SA.speed = max(0.6, SA.speed-0.4)
+	else
+		dismount()
+
+/spell/mountup/proc/dismount(kind, amount)
+	var/mob/living/user = src.holder
+	playsound(user, 'sound/voice/cow.ogg', 100, 1)
+	user.overlays.Remove(my_overlay)
+	user.unregister_event(/event/damaged, src, .proc/dismount)
+	active = FALSE
+	if(istype(user,/mob/living/simple_animal))
+		var/mob/living/simple_animal/SA = user
+		SA.speed = remembered_speed
+
+/obj/effect/overlay/horsebroom_mount
+	name = "steed"
+	icon = 'icons/mob/in-hand/left/items_lefthand.dmi'
+	icon_state = "horsebroom0"
+	layer = VEHICLE_LAYER
+	plane = ABOVE_HUMAN_PLANE
```

Comment:
Instead of using `relative_plane()` as I mentioned above, you could also just change this to just directly be a relative plane. Like just `FLOAT_PLANE + 1`. Or I guess `+ 2` if you want it to be the same plane difference it would be with `relative_plane()`.